### DISCUSSION TITLE
Adding terms for the dictionary

### DISCUSSION THREAD

**Discussion by nightcoffeebean** [xicri/genshin-dictionary#134] [General]
Hello, is there a way to submit or request terms that are not yet in the dictionary? Like the Primordial One, Phanes, etc

**Comment by xicri** [comment]
Hi, @nightcoffeebean,

Thanks for considering contribution to Genshin Dictionary.

The translation data is managed in xicri/genshin-langdata repository, so please send me a Pull Request on the repo.
Here's development guide. Let me know if you have any question. (Unfortunately, some docs are only in Japanese currently.)
https://github.com/xicri/genshin-langdata?tab=readme-ov-file#development

Sorry, since v4.1, I'm not actively updating the Dictionary (temporarily).
However, @SleepyAsh0191 is working on adding new terms, so please let him know when you start working to avoid duplicate works or conflict.
